docs/source/reference/importance.rst

```diff
@@ -0,0 +1,12 @@
+.. module:: optuna.importance
+
+Hyperparameter Importance
+=========================
+
+.. autofunction:: get_param_importances
+
+.. autoclass:: BaseImportanceEvaluator
```

Comment:
**FYI** [BaseSampler](https://optuna.readthedocs.io/en/stable/reference/samplers.html#optuna.samplers.BaseSampler) is exposed in the doc.

Reply:
@sile 
Thanks for your comment. Isn't it because `BaseSampler` is supposed to be used by a library user?
Either is actually fine for me, but I just think we should be consistent.

Reply:
I basically agree with @g-votte option. Because, unlike `BaseSampler`, it's not expected that users wll implement their own custom importance evaluators, I think that it's not needed to expose `BaseImportanceEvaluator ` to users at the time being.

Reply:
Fixed!